sprint_2/Product.js

```diff
@@ -0,0 +1,51 @@
+class Product {
+  // 필드 선언 (값은 나중에 constructor에서 할당)
+  name;
+  description;
+  price;
+  tags;
+  images;
+  favoriteCount;
+
+  constructor(name, description, price, tags, images, favoriteCount = 0) {
+    this.name = name;
+    this.description = description;
+    this.price = price;
+    this.tags = tags;
+    this.images = images;

```

Comment:
참조타입(객체, 배열, 함수 등)은 그대로 넣으면 외부에서 tags를 변경하면 객체 내부도 같이 변경됩니다. 따라서 복사하여 넣어주어야합니다.
```suggestion
    this.tags = [...tags];
    this.images = [...images];

```